react/components/Telemarketing.js

```diff
@@ -33,10 +33,14 @@ class Telemarketing extends Component {
           isLogged ? 'bg-emphasis' : 'bg-base--inverted'
           } z-999 pa2`}
       >
-        <div className="flex items-center">
+        <div className="flex items-center w-50">
           <TelemarketingIcon />
 
-          {!mobile && (
+          {mobile ? (
```

Comment:
To avoid code duplication, use this condition inside the div to decide what to render.